data/json/effects.json

```diff
@@ -460,6 +481,34 @@
     },
     {
         "type": "effect_type",
+         "id": "ooze_in_eyes",
+        "name": ["Gunk in eyes"],
+        "desc": ["There is some sort of substance in your eyes. You can barely see anything now!"],
+        "apply_message": "Your eyes sting and you can keep them barely open!",
+        "rating": "bad",
+        "base_mods": {
+            "per_mod": [-3]
+        }
+    },
+    {
+        "type": "effect_type",
+        "id": "inked",
+        "name": ["Soaked in ink"],
+        "desc": ["You are covered in black ink!"],
+        "apply_message": "You're covered in black ink!",
+        "rating": "good",
+        "max_intensity": 100,
```

Comment:
That max intensity is huge.
It should probably only allow negating enough scent to counter regular smell (500? I don't remember - the scent calculation method has it all).
The Strong Scent trait (and its advanced form - Smelly) should be only partially affected by scent masking, and that shouldn't be hardcoded, but balanced by lowering the max intensity (or scaling) here.